ecocode-rules-specifications/src/main/java/io/ecocode/rules/Common.java

```diff
@@ -0,0 +1,43 @@
+package io.ecocode.rules;
+
+import org.sonar.api.SonarEdition;
+import org.sonar.api.SonarProduct;
+import org.sonar.api.SonarQubeSide;
+import org.sonar.api.SonarRuntime;
+import org.sonar.api.utils.Version;
+
+/**
+ * This class is used to reduce duplication code and to pass SonarCloud Quality Gate
+ * //TODO: Remove this class when repository will be split
+ */
+public final class Common {
+  private static final Version SONARQUBE_RUNTIME_VERSION = Version.create(9, 8);
+
+  /**
+   * Base compatibility version of plugins
+   */
+  public static final SonarRuntime SONARQUBE_RUNTIME = new SonarRuntime() {
```

Comment:
We don't need to recreate an instance of SonarRuntime object. This one will be auto-injected by SonarQube in the constructor of repositories. You can find an example here: https://github.com/green-code-initiative/ecoCode/pull/201/files#diff-2e1bd8296a7edbbc1cbd42040b52bb2c1fa138955568bf12938368d04cd81545R44-R48

Reply:
I changed implementation